code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -411,6 +438,7 @@
 	description = "A commercial chemical designed to help older men in the bedroom." //not really it just makes you a giant
 	color = "#ff0000"//strong red. rgb 255, 0, 0
 	var/current_size = 1
+	taste_message = "power!"
```

Comment:
will the punctuation cause an issue with the existing punctuation in the rest of the message?